app/services/zendesk_intake_service.rb

```diff
@@ -164,7 +164,7 @@ def send_final_intake_pdf
       Online intake form submitted and ready for review. The taxpayer was notified that their information has been submitted. (automated_notification_submit_confirmation)
 
       Client's provided interview preferences: #{@intake.interview_timing_preference}
-
+      #{"Client is filing for Economic Impact Payment support" if @intake.filing_for_stimulus_yes?}
```

Comment:
This was my best guess for where the requirement in the story: "Should add a comment with the 13614-C "client is filing for Economic Impact Payment support". It may very well be that something else was intended?

Reply:
let's add it to the first place where we upload the 13614c also